src/main/java/net/minecraftforge/client/settings/KeyModifier.java

```diff
@@ -158,7 +170,7 @@ public static KeyModifier valueFromString(String stringValue)
 
     public abstract boolean matches(int keyCode);
 
-    public abstract boolean isActive();
+    public abstract boolean isActive(@Nullable IKeyConflictContext conflictContext);
```

Comment:
Ah I completely over looked that. I'll add `isActive()` back and deprecate it. Nothing can implement `KeyModifier` except this class so it will no longer be a breaking change.